packages/ui-components/src/components/layout/Dialog.vue

```diff
@@ -104,23 +119,26 @@
 <script setup lang="ts">
 import { Dialog, DialogPanel, TransitionChild, TransitionRoot } from '@headlessui/vue'
 import { FormButton, type LayoutDialogButton } from '~~/src/lib'
-import { XMarkIcon } from '@heroicons/vue/24/outline'
+import { XMarkIcon, ChevronLeftIcon } from '@heroicons/vue/24/outline'
 import { useResizeObserver, type ResizeObserverCallback } from '@vueuse/core'
 import { computed, ref, useSlots, watch, onUnmounted } from 'vue'
 import { throttle } from 'lodash'
 import { isClient } from '@vueuse/core'
 
-type MaxWidthValue = 'sm' | 'md' | 'lg' | 'xl'
+type MaxWidthValue = 'sm' | 'md' | 'lg' | 'xl' | 'fullscreen'
```

Comment:
Why is this necessary? IIRC if you omit `maxWidth` it's already full-screen

Reply:
@fabis94 If there is no `maxWidth` the width is set to `2xl:max-w-7xl`, in addition `fullscreen` always make the modal full screen (max height as well), maybe makes more sense to change it to a `fullscreen` prop?

Reply:
@Mikehrn I see. In that case, I agree that there should just be a `fullscreen?: boolean` prop that enables this full-width full-height behavior.